opm/autodiff/SimulatorFullyImplicitBlackoilOutput.cpp

```diff
@@ -296,8 +296,21 @@ namespace Opm
             }
             else {
                 // just write the data to disk
-                writeTimeStepSerial( timer, state, wellState, cellData, substep );
+                int err = 0;
+                try {
+                    writeTimeStepSerial( timer, state, wellState, cellData, substep );
+                    MPI_Bcast(&err, 1, MPI_INT, 0, MPI_COMM_WORLD);
+                } catch (std::runtime_error& msg) {
```

Comment:
yes, it's hard to make it much more readable as only one process shall write, and i need to catch the exception, communicate status to other processes, then rethrow. i can add a bool and copy the error message out of the exception, then rethrow at another point so we only get a single bcast call if you think that is better.